avogadro/rendering/geometryvisitor.h

```diff
@@ -61,7 +71,34 @@ class GeometryVisitor : public Visitor
    * Get the radius of the scene.
    */
   float radius();
+#ifdef _3DCONNEXION
```

Comment:
Again, this seems like a perfectly reasonable API addition without the `#ifdef` - and thus the implementation should be in the relevant `.cpp` file.